packages/docfd/docfd.7.0.0/opam

```diff
@@ -0,0 +1,82 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "TUI multiline fuzzy document finder"
+description: """
+
+Think interactive grep for text files, PDFs, DOCXs, etc,
+but word/token based instead of regex and line based,
+so you can search across lines easily.
+
+Docfd aims to provide good UX via integration with common text editors
+and PDF viewers,
+so you can jump directly to a search result with a single key press.
+
+Features:
+
+- Multithreaded indexing and searching
+
+- Multiline fuzzy search of multiple files or a single file
+
+- Swap between multi-file view and single file view on the fly
+
+- Content view pane that shows the snippet surrounding the search result selected
+
+- Text editor and PDF viewer integration
+              """
+maintainer: ["Darren Li"]
+authors: ["Darren Li"]
+license: "MIT"
+tags: ["fuzzy" "document" "finder"]
+homepage: "https://github.com/darrenldl/docfd"
+doc: "https://github.com/darrenldl/docfd"
+bug-reports: "https://github.com/darrenldl/docfd/issues"
+depends: [
+  "ocaml" {>= "5.2"}
+  "dune" {>= "3.4"}
+  "fmt" {>= "0.9.0"}
+  "angstrom" {>= "0.15.0"}
+  "containers" {>= "3.12"}
+  "oseq"
+  "spelll"
+  "notty"
+  "nottui" {= "0.3"}
+  "lwd"
+  "cmdliner" {>= "1.1.0"}
+  "eio" {>= "0.14"}
+  "digestif"
+  "yojson" {>= "2.0.2"}
+  "eio_main"
+  "containers-data"
+  "timedesc" {>= "2.0.0"}
+  "re" {>= "1.11.0"}
+  "ppx_deriving" {>= "5.0"}
+  "alcotest" {with-test}
+  "qcheck-alcotest" {with-test}
+  "qcheck" {with-test}
+  "odoc" {with-doc}
+]
+dev-repo: "git+https://github.com/darrenldl/docfd.git"
+build: [
+  ["dune" "subst"] {dev}
+  [
+    "dune"
+    "build"
+    "-p"
+    name
+    "-j"
+    jobs
+    "@install"
+    "@doc" {with-doc}
+  ]
+]
+pin-depends: [
+  [ "nottui.0.3" "git+https://github.com/let-def/lwd.git#a337a778001e6c1dbaed7e758c9e05f300abd388" ]
+]
```

Comment:
As flagged by our linting rules, `pin-depends` are not permitted in the repo. Do you think you could work on getting this merged into nottui first?

Reply:
I know someone is working on a nottui fork, I'll wait for that first then. Thanks.